cypress/e2e/shared/editor.test.ts

```diff
@@ -91,12 +94,33 @@ describe('Editor+LSP communication', () => {
   })
 
   describe('in Script Editor', () => {
+    const setScriptToFlux = () => {
+      return cy.isIoxOrg().then(isIox => {
```

Comment:
On iox, there is an ability to toggle between SQL and flux.

<img width="298" alt="Screen Shot 2022-12-27 at 8 09 52 PM" src="https://user-images.githubusercontent.com/10232835/209742138-b9c2d3d5-056a-4234-809d-49a205f82df7.png">

This `setScriptToFlux()` ensures we are on the flux language.